exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,9 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
+  return number >= 0 ? "positive" : "negative";
+}
```

Comment:
Nice use of ternary operator! 👍  You could also write this function in one line by declaring it as an arrow function, you wouldn't even need to use `return` and curly brackets  :)